src/Sulu/Bundle/CategoryBundle/Api/Category.php

```diff
@@ -348,6 +356,17 @@ public function getTranslation()
                 return $translation;
             }
         }
+
+        return;
+    }
+
+    /**
+     * Creates a new Translation for category.
+     *
+     * @return CategoryTranslation
+     */
+    public function createTranslation()
```

Comment:
Would you make this method public? I think it should only be used inside this class.